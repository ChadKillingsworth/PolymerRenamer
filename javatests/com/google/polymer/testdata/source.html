<!doctype html>
<html>
<head>
<title>
</title>
</head>
<body>
<!-- No renames in the next script tag since it's not in polymer-element. -->
<script>
  var foo = my.coolSymbolName;
  foo();
</script>
<polymer-element
    name="test"
    attributes="MY_SYMBOL prop1 coolSymbolName prop2 symbolA">
  <template>
    <style>
      symbolA {
      }
    </style>
    <div id="{{foo.dontRenameMe}}" name="{{foo.coolSymbolName}}" selected="0" foo?="{{polymerdoesthis}}">
      {{symbolA}}
    </div>
    <div selected>
      Another selected div.
    </div>
    <template if="{{symbolA == 42}}">
      symbolA is 42.
    </template>
  </template>
  <script>
    Polymer({
      symbolAChanged: function() {this.coolSymbolName = true;},
      prop1Changed: function() {this.prop2 = 42;}
    });
  </script>
</polymer-element>
</body>
</html>
