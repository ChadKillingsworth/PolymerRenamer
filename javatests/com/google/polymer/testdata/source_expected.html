<!doctype html><html><head>
<title>
</title>
</head>
<body>
<!-- No renames in the next script tag since it's not in polymer-element. -->
<script>
  var foo = my.coolSymbolName;
  foo();
</script>
<polymer-element name="test" attributes="cd3 prop1 b prop2 ab">
  <template>
    <style>
      symbolA {
      }
    </style>
    <div id="{{foo.dontRenameMe}}" name="{{foo.b}}" selected="0" foo?="{{polymerdoesthis}}">
      {{ab}} [[symbolA]] [[foo.dontRenameMe]]
    </div>
    <div selected>
      Another selected div.
    </div>
    <template if="{{ab==42}}">
      symbolA is 42.
    </template>
  </template>
  <script>Polymer({abChanged:function(){this.b=true},prop1Changed:function(){this.prop2=42}});</script>
</polymer-element>

</body></html>
